irods_specific_implementation/utils.hpp

```diff
@@ -63,6 +64,14 @@ namespace irods::rest
 
         _response.send(http_code, msg);
     } // handle_request
+
+    inline bool http_bool(const std::string& s) noexcept
+    {
+        return s == "1" || s == "true";
+    }
```

Comment:
This function appears to be defined multiple times.

The use of `inline` is justified here since this is a header file.